templates/rubocop.yml

```diff
@@ -68,5 +68,7 @@ Style/ClassAndModuleChildren:
   Enabled: false
 Rails/TimeZone:
   Enabled: false
+Rails:
```

Comment:
Did you rebase with master? Since I have this in Master already. So no need to add it here again